libs/api/domains/auth/src/lib/dto/delegationScope.input.ts

```diff
@@ -1,10 +1,15 @@
 import { Field, InputType } from '@nestjs/graphql'
 
+import { ScopeType } from '@island.is/clients/auth-public-api'
+
 @InputType('AuthDelegationScopeInput')
 export class DelegationScopeInput {
   @Field((_) => String)
   name!: string
 
+  @Field((_) => ScopeType)
```

Comment:
@darrikonn What does the (_) stand for

Reply:
It's basically an unused parameter. So Field expects 1 argument inside its function, but that argument would be unused, so we bypass the linting error by naming that parameter `_`, `_0`, `_1`, `_*`.

```json
        "@typescript-eslint/no-unused-vars": [
          "warn",
          {
            "ignoreRestSiblings": true,
            "argsIgnorePattern": "^_([0-9])?$",
            "varsIgnorePattern": "^_([0-9])?$"
          }
        ],

```